### PR TITLE
Add: hover function is fired to the active element

### PR DESCRIPTION
**Pull request by cicarulez**

### CODE REVIEW

**Comment by cicarulez** [comment]
Please update TypeDefinition

File: src/defaults.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ let defaults = {
       fillColor: 'rgba(200, 200, 200, 0.75)', // optional: custom background color for item
       content: 'a command name' // html/text content to be displayed in the menu
       contentStyle: {}, // css key:value pairs to set the command's css in js if you want
+      hover: function(ele){ // a function to execute when the command is hovered
+        console.log( ele.id() ) // `ele` holds the reference to the active element
+      },
```
Comment (maxkfranz) [inline src/defaults.js:12]: Would you add your new option to the documentation in the readme?

**Comment by maxkfranz** [comment]
It looks like it's working correctly.

@cicarulez, would you add your new option in the documentation in the readme?  I noticed that it doesn't activate the `hover` function for disabled menu items.  That may be the most sensible option, but that edge case should be documented in the readme so that users of the extension aren't surprised by how it works.

**Comment by maxkfranz** [comment]
@cicarulez, are you still interested in adding the readme documentation to this PR for your feature?

**Pull request merged**